what's new.md

```diff
@@ -465,4 +465,5 @@ BABYLON.SceneLoader.OnPluginActivatedObservable.add(function(plugin) {
 ```
 - `currentState` and `previousState` have been removed from use in `onInputChangedObservable` in the `DeviceSourceManager` ([PolygonalSun](https://github.com/PolygonalSun))
 - `PointerInput` movement enums are no longer being used in any movement event handling in the `DeviceInputSystem` and `InputManager` ([PolygonalSun](https://github.com/PolygonalSun))
-- Shadow generators now use the `Material.alphaCutOff` value instead of a hard-coded 0.4 value. ([Popov72](https://github.com/Popov72))
\ No newline at end of file
+- Shadow generators now use the `Material.alphaCutOff` value instead of a hard-coded 0.4 value. ([Popov72](https://github.com/Popov72))
+- `PointerInput` mouse enums are no longer usable with the `DeviceSource`'s `getInput` function for Mouse and Touch ([PolygonalSun](https://github.com/PolygonalSun))
```

Comment:
I don't think we have a new one for future releases yet. Maybe need to check with @RaananW.

Reply:
https://github.com/BabylonJS/Babylon.js/pull/12397#issuecomment-1099698950

No need to update the what's new file, just make sure the PR is tagged correctly.

Reply:
The change log can be found at CHANGELOG.md on the root directory

Reply:
File reverted.

Reply:
Okay, we should make sure the PR titles are good then since it appears that ends up in the change log.